src/avlizer.app.src

```diff
@@ -1,6 +1,6 @@
 {application, avlizer,
  [{description, "Avro Serializer"},
-  {vsn, "0.3.4"},
+  {vsn, "0.3.5"},
```

Comment:
Since this PR adds a new feature, it deserves a minor version bump.

Reply:
Done :+1: